.travis.yml

```diff
@@ -1,15 +1,21 @@
 language: node_js
 sudo: false
 node_js:
-  - 0.10
+- '0.10'
 before_script:
-  - npm rebuild
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
+- npm rebuild
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
 script: npm test
+after_success: npm run-script publish-release > /dev/null 2>&1
 notifications:
   irc:
     channels:
-      - "irc.mozilla.org#payments"
+    - irc.mozilla.org#payments
     on_success: change
     on_failure: always
+env:
+  global:
```

Comment:
could you maybe add a comment about whose account these variables are defined in?